src/paperless/settings.py

```diff
@@ -90,9 +95,10 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "..", "data", "db.sqlite3"),
+        "NAME": os.path.join(os.getenv("PAPERLESS_DBLOCATION",os.path.join(BASE_DIR, "..", "data")), "db.sqlite3")
```

Comment:
Strike that, my suggestion would be `PAPERLESS_DBDIR` to remain consistent with the other _DB* names.  I just think that "DIR" should be there just to make it obvious.